main.go

```
@@ -59,8 +83,8 @@ func main() {
 	kingpin.HelpFlag.Short('h')
 	kingpin.Parse()
 
-	config = loadConfig(*configFile)
-	createSnClient(config)
+	loadConfig(*configFile)
```

Wouldn't it be better to return an error and centralize the error management in the main method ?

---

Even though this was not a change made in the PR, I will take a look.

---

Done.